README.md

```diff
@@ -6,72 +6,12 @@ Formally Verified Vyper Contracts
 FVyper is a collection of formally verified, useful Vyper programs.  
 See [roadmap](https://github.com/LayerXcom/verified-vyper-contracts/issues/5).
 
-## How to get started
+## Structure of this project
```

Comment:
`Directory structure` ?

Reply:
`Directory structure` is easy to understand, OK!